config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1109,7 +1109,7 @@ presubmits:
             - --deployment=node
             - --gcp-zone=us-central1-b
             - --node-args=--image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/image-config-serial-resource-managers.yaml
-            - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --cgroup-driver=systemd"
+            - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --cgroup-driver=systemd" --service-feature-gates="PodLevelResources=true" --feature-gates="PodLevelResources=true"
```

Comment:
should `--feature-gates="PodLevelResources=true"` be part of the kubelet flags?

Reply:
kubelet feature gates are calculated from `--feature-gates`, no need to pass into kubelet-flags